openrndr-math/src/main/kotlin/org/openrndr/math/Vector3.kt

```diff
@@ -79,7 +79,7 @@ data class Vector3(val x: Double, val y: Double, val z: Double) : Serializable,
     infix fun projectedOn(v: Vector3) = (this dot v) / (v dot v) * v
 
     /** The Euclidean length of the vector. */
-    val length: Double get() = Math.sqrt(x * x + y * y + z * z)
+    val length: Double get() = sqrt(x * x + y * y + z * z)
```

Comment:
In general I guess due to the fact that folders where moved in math module to adhere to mpp conventions, some of these changes are either already there, or will have to be resolved due to conflict.

Reply:
I think it makes sense to see this PR as a reminder of things that need to be resolved. The problem with this PR is that it is huge and not easily verified. While I don't think it will break things (and definitely not willingly) I am not very eager to merge it in. This is not to discredit @hamoid 's work, they have indicated themselves that the monolithic nature of the PR may be problematic.

Reply:
Yeah I take on offense :-) I could split this in smaller PRs when I have some time.